src/encoded/types/biosample.py

```diff
@@ -832,6 +847,9 @@ def generate_sentence(phrases_dict, values_list):
             if 'preservation_method' in key:
                 sentence = sentence.strip() + ', ' + \
                                     phrases_dict[key].strip() + ' '
+            elif 'post_nucleic_acid_delivery_time' in key:
```

Comment:
As-is this works so it is fine. But I would recommend f-string (https://cito.github.io/blog/f-strings/) for the future.